pkg/util/helpers_test.go

```diff
@@ -22,10 +22,57 @@ under the License.
 package util
 
 import (
+	"reflect"
 	"sort"
 	"testing"
 )
 
+func TestMergeEnvMaps(t *testing.T) {
+	var tests = []struct {
+		source      map[string]string
+		destination map[string]string
+		expected    map[string]string
+	}{
+		// case : nothing is done for empty
```

Comment:
Can you just add a field to the struct `description` and move all the comment text into that field?  It would help if in future any of these test cases break.  Otherwise looks great!